Commands/Commands/uservoice.js

```diff
@@ -261,6 +261,8 @@ commands.info = {
           }).then(response => {
             latestSuggLink = response.suggestions[0].url
             latestSuggTitle = response.suggestions[0].title
+            let roboDate = new Date(data.user.created_at)
+            let humanDate = roboDate.getUTCFullYear() + '/' + roboDate.getUTCMonth() + '/' + roboDate.getUTCroboDateay() + ' at ' + roboDate.getUTCHours() + ':' + roboDate.getUTCMinutes() + ':' + roboDate.getUTCSeconds()
```

Comment:
Since moment is now one of our deps, probably want to make this a bit nicer (I'll do it)